src/ipc/types.ts

```diff
@@ -3,6 +3,8 @@ import { ItemCheckWidget, PriceCheckWidget } from '@/web/overlay/interfaces'
 export interface Config {
   configVersion: number
   leagueId?: string
+  privateLeagueName?: string
```

Comment:
I'd reuse `leagueId`. You can know if it's private if it has (PL???) at the and, OR if it is not in the list of public leagues.

Reply:
The variable if something to store the textbox value in, leagueId is actually the selected leagueId

Reply:
> copy paste the name the whole time

You can use dropdown to temporarily change and price-check in Tmp or Std league. So this use-case is covered. 

If you want to often perma switch between public and private league I think it defeats the point of private league, just play in public wtf

Reply:
Makes sense, ill remove it